layouts/partials/head.html

```diff
@@ -77,7 +77,7 @@
 <script src="{{ "js/jquery-ui-1.12.1.min.js" | relURL }}"></script>
 <script src="{{ "js/sweetalert-2.1.2.min.js" | relURL }}"></script>
 
-{{ if eq .Params.mermaid true }}
+{{ if .HasShortcode "mermaid" }}
```

Comment:
@sftim, This is a nice change. In reviewing the changes (current page on master and this proposed change), I noticed
that upon page reload, the mermaid
markup displays before the image is repainted. Do you know if there is a way
to turn off the display of the mermaid markup? I don't see the markup text in the display of the mermaid examples.

Reply:
I haven't really looked into how MermaidJS works: I'm at the level of “if you have JavaScript off you don't get diagrams, and if you have it on, you do”. I've used PlantUML a bit though so I know how to author for it.

This change doesn't affect the MermaidJS behavior, it just saves you having to remember to update the front matter if you add / remove a MermaidJS shortcode.

Reply:
Right, I understand, though you added mermaid diagrams to this page. The diagrams render OK, though, the
reload of the page is not graceful.

Reply:
_Aside_
It's _technically feasible_ to render the Mermaid source to .svg during static site generation, and that would be a nice optimization. However: it's also lots of work.